src/transmitter/payload.ts

```diff
@@ -134,3 +139,49 @@ export function constructWorkloadEventsPolicy(
     agentId: config.AGENT_ID,
   };
 }
+
+const workloadKindMap = {
+  deployment: 'Deployment',
+  replicaset: 'ReplicaSet',
+  statefulset: 'StatefulSet',
+  daemonset: 'DaemonSet',
+  job: 'Job',
+  cronjob: 'CronJob',
+  replicationcontroller: 'ReplicationController',
+  deploymentconfig: 'DeploymentConfig',
+  pod: 'Pod',
+};
+export function constructRuntimeData(
+  runtimeResults: IRuntimeImage[],
+): IRuntimeDataPayload {
+  const filteredRuntimeResults = runtimeResults.reduce((acc, runtimeResult) => {
+    if (!isExcludedNamespace(runtimeResult.namespace)) {
+      const mappedWorkloadKind =
+        workloadKindMap[runtimeResult.workloadKind.toLowerCase()];
+      if (mappedWorkloadKind) {
+        runtimeResult.workloadKind = mappedWorkloadKind;
+        acc.push(runtimeResult);
+      } else {
+        logger.error({ runtimeResult }, 'invalid Sysdig workload kind');
```

Comment:
Nit: I would probably remove the `packages[]` from the `runtimeResult` when logging it to avoid really large logs